field_encrypt.module

```diff
@@ -45,20 +145,22 @@ function field_encrypt_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $
  * @param \Drupal\Core\Form\FormStateInterface $form_state
  */
 function field_encrypt_form_field_add_form_builder($entity_type, \Drupal\field\Entity\FieldStorageConfig $fieldStorageConfig, &$form, \Drupal\Core\Form\FormStateInterface $form_state) {
-
-  $form_encryption = $form_state->getValue('encrypt');
-  $original_encryption = $fieldStorageConfig->getThirdPartySetting('field_encrypt', 'encrypt');
+  $field_encryption_settings = $form_state->getValue('field_encrypt');
+  $form_encryption = $field_encryption_settings['encrypt'];
+  $original_encryption = $fieldStorageConfig->getThirdPartySettings('field_encrypt');
```

Comment:
After this line, I would return if you do not have any settings enabled for field encrypt.